maintainers/team-list.nix

```diff
@@ -41,6 +46,7 @@ with lib.maintainers; {
       ylecornec
     ];
     scope = "Bazel build tool & related tools https://bazel.build/";
+    featureFreezeTeamName = "Bazel";
```

Comment:
Hm, but setting the bool requires setting a team name as well. I don't really see a point in having both since you will always have to set the string and also another bool that is always `true` when the name is set.

Reply:
But now every other script that follows will have to use the `featureFreezeTeamName` if they want a pretty name. Or they have to refactor the structure another time.

Reply:
Is this better? I also changed `scope` to `description` because it's a pattern I've seen more often in nixpkgs than `scope`

Reply:
I would've just made the human-readable name mandatory. Setting it is a small ask for everyone touching this file.